deployment/stacks/service/base/template/proxy/uec-dos-api-cs-proxy-ingress.yaml

```diff
@@ -10,8 +10,10 @@ metadata:
     alb.ingress.kubernetes.io/backend-protocol: HTTPS
     alb.ingress.kubernetes.io/healthcheck-protocol: HTTPS
     alb.ingress.kubernetes.io/ssl-policy: ELBSecurityPolicy-TLS-1-2-2017-01
+    alb.ingress.kubernetes.io/certificate-arn: arn:aws:acm:eu-west-2:AWS_ACCOUNT_ID_TO_REPLACE:certificate/AWS_CERTIFICATE_TO_REPLACE
+    alb.ingress.kubernetes.io/waf-acl-id: WAF_ACL_ID_TO_REPLACE
     alb.ingress.kubernetes.io/tags: Service=SERVICE_TAG_TO_REPLACE
-    alb.ingress.kubernetes.io/listen-ports: '[{"HTTPS":443}]'
+    alb.ingress.kubernetes.io/listen-ports: '[{"HTTPS":443},{"HTTP":80}]'
```

Comment:
At the moment if you go to the site via HTTP a CSRF cookie error is raised and so this redirect does not occur. Do we want to keep it in?